README.md

```diff
@@ -6,29 +6,41 @@
 
 ## Set up wallet
 
-1. Create Payment Keys\
-`$ cardano-cli address key-gen --verification-key-file payment.vkey --signing-key-file payment.skey`
-2. To generate a stake key pair\
-`$ cardano-cli stake-address key-gen --verification-key-file stake.vkey --signing-key-file stake.skey`
-3. Create Wallet Address\
-`$ cardano-cli address build --payment-verification-key-file payment.vkey --stake-verification-key-file stake.vkey --out-file payment.addr (--mainnet | --testnet-magic NATURAL)`
-4. Request Test ADA at [https://docs.cardano.org/cardano-testnet/tools/faucet](https://docs.cardano.org/cardano-testnet/tools/faucet)\
+There are two ways to setup a wallet. You can use your mnemonic to setup an already existing wallet or create a new wallet.
```

Comment:
I would say there are 3 options

- Restore wallet using mnemonics for the majority
- Create new wallet using `cardano-cli address key-gen...`
- Restore a wallet using existing keys (without mnemonics). Most stake pool operators use priv/public keys.

Reply:
Fixed on 5656f29